### ISSUE TITLE
Type analysis does not work as expected in debug mode.

### ISSUE THREAD

**Issue by STofone** [DetachHead/basedpyright#1402]
### Description

when debugging extension ( use server.js), the built-in type was not recognized. 
eg.
```shell
"str" is not definedbasedpyright[reportUndefinedVariable](https://docs.basedpyright.com/v1.30.1/configuration/config-files/#reportUndefinedVariable)
(function) str: Unknown

"classmethod" is not definedbasedpyright[reportUndefinedVariable](https://docs.basedpyright.com/v1.30.1/configuration/config-files/#reportUndefinedVariable)
(function) classmethod: Unknown
```

**Comment by STofone** [comment]
While it just works fine using pyright executable

**Comment by STofone** [comment]
I forgot to copy `typeshed-fallback` into the directory.

There is no error reported in log whitout it.

**Comment by DetachHead** [comment]
you shouldn't have to manually copy anything over when debugging. are you following the building & debugging instructions documented [here](https://docs.basedpyright.com/latest/development/contributing/#building)?

**Comment by STofone** [comment]
> you shouldn't have to manually copy anything over when debugging. are you following the building & debugging instructions documented [here](https://docs.basedpyright.com/latest/development/contributing/#building)?

I am developing my own extension. Some build options are different.
